Libraries/Microsoft.Teams.Apps/App.cs

```diff
@@ -172,7 +172,7 @@ await Events.Emit(
     /// send an activity to the conversation
     /// </summary>
     /// <param name="activity">activity activity to send</param>
-    public async Task<T> Send<T>(string conversationId, T activity, ConversationType? conversationType = null, string? serviceUrl = null, CancellationToken cancellationToken = default) where T : IActivity
+    public async Task<T> Send<T>(string conversationId, T activity, ConversationType? conversationType = null, string? serviceUrl = null, bool isTargeted = false, CancellationToken cancellationToken = default) where T : IActivity
```

Comment:
Instead of updating the method signature, can you create an overload for this? As is, it's currently a breaking change:

Customers using the method this way will not work anymore:
```cs
app.Send(convId, activity, convType, serviceUrl, cancellationToken)
```
and will have to be updated to include the isTargetted boolean in args list.